src/js/app.js

```diff
@@ -0,0 +1,49 @@
+const table = document.querySelector('.table');
+// table.addEventListener('click', tableOnClick);
+let colIndex = -1;
+
+function sortTable(index, type, isSorted) {
+    const tbody = table.querySelector('tbody');
```

Comment:
bring that variable above because every time you call a function, you recreate it. PERFORMANCE